src/etc/cargo.bashcomp.sh

```diff
@@ -246,7 +246,7 @@ _get_examples(){
 	local manifest=$(_locate_manifest)
 	[ -z "$manifest" ] && return 0
 
-	local files=("${manifest%/*}"/examples/*.rs)
+	local files=("${manifest%/*}"/examples/**/*.rs)
```

Comment:
This could pick up both examples and submodules.  Unsure whether that is worth it or not

Reply:
I would be a little worried that the false-positive rate would be too high. In bevy for example, this would auto-complete a ton of names that aren't examples.

I think the only real solution here is to parse Cargo.toml, and I think native-completions is the only way to realistically do that. This bashcomp could do some json parsing of `cargo metadata`, but I wouldn't want to go there.